backend/rhapp_aws_s3-main/app.py

```diff
@@ -0,0 +1,55 @@
+from boto3 import client, resource
+from credentials import ACCESS_KEY, SECRET_ACCESS_KEY
+import matplotlib.pyplot as plt
+import matplotlib.image as mpimg
+import io
+from PIL import Image, ImageGrab
+
+user = client(
+    's3',
+    aws_access_key_id=ACCESS_KEY,
+    aws_secret_access_key=SECRET_ACCESS_KEY,
+    region_name='ap-southeast-1'
+)
+
+res = resource(
+    's3',
+    aws_access_key_id=ACCESS_KEY,
+    aws_secret_access_key=SECRET_ACCESS_KEY,
+    region_name='ap-southeast-1'
+)
+
+bucket = res.Bucket('rhapp-picture-bucket')
+
+#create
+def create(file):
+    obj = bucket.Object(file)
+    img = Image.open('stickman.png')
```

Comment:
1. Change file arugment to key, makes more sense as the key in the bucket
2. Also take in an argument for fileLocation to indicate where the local file is

Reply:
Before creation, check if a file with key already exist. If so throw a conflict error. If not, may cause accidental deletion of files